app/frontend/models/user.ts

```diff
@@ -58,6 +58,18 @@ export const UserModel = types
       const response = yield self.environment.api.restoreUser(self.id)
       return response.ok
     }),
+    changeRole: flow(function* () {
+      if (self.role === EUserRoles.reviewManager) {
+        var newRole = EUserRoles.reviewer
+      } else if (self.role === EUserRoles.reviewer) {
+        var newRole = EUserRoles.reviewManager
```

Comment:
change `var` to let and declare outside of if blocks, to keep consistent with modern javascript